ArduCopter/AP_Arming.h

```diff
@@ -46,6 +46,7 @@ class AP_Arming_Copter : public AP_Arming
     // NOTE! the following check functions *DO NOT* call into AP_Arming!
     bool parameter_checks(bool display_failure);
     bool motor_checks(bool display_failure);
+    bool motor_count_check(void);
```

Comment:
As @WickedShell points out, a count is insufficient, so this would need renaming.